### ISSUE TITLE
Resist Fire buff

### ISSUE THREAD

**Issue by Plasmashield** [azerothcore/azerothcore-wotlk#9283]
### Current Behaviour

https://wowgaming.altervista.org/aowow/?spell=15123
Resist Fire, (u can buff it by using mind control) changes the value to 2 FR after relog

### Expected Blizzlike Behaviour

The value should remain 2 + 1.5 per mob's level even after the relog.

### Steps to reproduce the problem

get buff
relog

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/00dc369cb627d728c520cd597b31e9edf2c75eed

### Operating system

Win10

**Comment by Plasmashield** [comment]
@Shineslippers True. Didn't notice that. I've seen it on the CC, but every time im tp and relog right after dungeon. So i thought bug appears only after relog. I'm sorry for this inaccuracy. Should i correct my report?

**Comment by Shineslippers** [comment]
No worry. That's my part to test it. Anyway devs will read my comment. Hm i can even do it by myself.

**Comment by Plasmashield** [comment]
Thank you!
